server/graphql/core/src/loader/sensor.rs

```diff
@@ -0,0 +1,32 @@
+use repository::EqualFilter;
+use repository::{
+    sensor::{Sensor, SensorFilter, SensorRepository},
+    RepositoryError, StorageConnectionManager,
+};
+
+use async_graphql::dataloader::*;
+use async_graphql::*;
+use std::collections::HashMap;
+
+pub struct SensorByIdLoader {
+    pub connection_manager: StorageConnectionManager,
+}
+
+#[async_trait::async_trait]
+impl Loader<String> for SensorByIdLoader {

```

Comment:
Btw, would need this only when it's a field in a collection, for example for locations